### PR TITLE
Group stx in block page

### PR DESCRIPTION
**Pull request by McEdward**

### CODE REVIEW

**Comment by chappjc** [comment]
looks good

File: dcrsqlite/apisource.go [hunk 1]
```diff
@@ -9,6 +9,8 @@ import (
 	"fmt"
 	"sync"
 
+	"github.com/decred/dcrd/blockchain/stake"
+
```
Comment (chappjc) [inline dcrsqlite/apisource.go]: please group imports from same site

File: dcrsqlite/apisource.go [hunk 2]
```diff
@@ -149,28 +151,29 @@ func (db *wiredDB) GetBlockVerboseByHash(hash string, verboseTx bool) *dcrjson.G
...
+		case stake.TxTypeSSRtx:
+			revocations = append(revocations, stx)
+		}
 	}
 	return &apitypes.BlockDataWithTxType{
 		blockVerbose,
-		stxTypes,
+		votes,
+		tickets,
+		revocations,
```
Comment (chappjc) [inline dcrsqlite/apisource.go]: Please tag each field in this literal defn:

```go
	return &apitypes.BlockDataWithTxType{
		GetBlockVerboseResult: blockVerbose,
		Votes:   votes,
		Tickets: tickets,
		Revs:    revocations,
	}
```

File: views/block.tmpl [hunk 3]
```diff
@@ -128,16 +128,15 @@
 
         <div class="row">
             <div class="col-md-12">
-                <h4 id="stake-transactions"><span>Stake Transactions</span></h4>
+                <h4 id="Vote"><span>Votes</span></h4>
```
Comment (chappjc) [inline views/block.tmpl]: I think lowercase `id`s are more idiomatic (`id="votes"`)

File: views/block.tmpl [hunk 4]
```diff
@@ -155,6 +153,59 @@
         </div>
 
         <div class="row">
+            <div class="col-md-12">
+                <h4 id="Vote"><span>Tickets</span></h4>
```
Comment (chappjc) [inline views/block.tmpl]: some copy pasta here.  `id="Vote"` --> `id="tickets"`

File: views/block.tmpl [hunk 5]
```diff
@@ -155,6 +153,59 @@
...
+                        </tr>
+                        {{end}}
+                    </tbody>
+                </table>
+            </div>
+        </div>
+        {{if .Revocations}}
+        <div class="row">
+            <div class="col-md-12">
+                <h4 id="Vote"><span>Revocations</span></h4>
```
Comment (chappjc) [inline views/block.tmpl]: `id="Vote"` --> `id="revocations"`

File: views/block.tmpl [hunk 6]
```diff
@@ -128,7 +128,7 @@
 
         <div class="row">
             <div class="col-md-12">
-                <h4 id="Vote"><span>Votes</span></h4>
+                <h4 id="vote"><span>Votes</span></h4>
```
Comment (chappjc) [inline views/block.tmpl]: `"vote"` -> `"votes"`

Please test links

Comment (McEdward) [reply views/block.tmpl]: alright

**Pull request merged**